changelog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog for [`process` package](http://hackage.haskell.org/package/process)
 
+## X.Y.Z
+
+* Add `System.Process.Env` and `System.Process.Env.OsString`. Bumps
+  `filepath >= 1.4.100.0`.
```

Comment:
This is now outdated. `s/Env/Environment/`?

Reply:
Good catch, thanks.